src/components/helper-tool.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { CSSProperties } from "react";
+import { XMarkIcon } from '@heroicons/react/20/solid'
+
+type HelperToolProps = {
+  data: Record<string, any>;
+  locale?: string;
+};
+
+export const HelperTool = ({ data, locale }: HelperToolProps) => {
+ if (!data){
+         return  <div></div>
+ }
+  const [isOpen, setIsOpen] = React.useState(true);
+  const { id, uid } = data;
+  const styling: CSSProperties = {
+    position: "fixed",
+    width: "150px",
+    background: "white",
+    right: 0,
+    bottom: 0,
+    zIndex: 100,
+    padding: 10,
+    border: "1px solid gray",
+    display: isOpen ? "block" : "none",
+  };
+
+  const editUrl = `https://www.yext.com/s/${data.businessId}/entity/edit3?entityIds=${uid}`;
+
+  const onCloseClick = () => {
+    setIsOpen(false);
+  };
+  
+  return (
+    <div className="yext-helper-tool" style={styling}>
+     <span className="isolate inline-flex rounded-lg shadow-sm">
+      <a
+      target="_blank"
```

Comment:
Add `rel="noopener noreferrer"`
https://stackoverflow.com/questions/50709625/link-with-target-blank-and-rel-noopener-noreferrer-still-vulnerable